src/components/form/form.types.ts

```diff
@@ -208,18 +215,21 @@ export interface FormComponentOptions {
 /**
  * @public
  */
-export interface FormLayoutOptions<T = FormLayoutType.Default> {
+export interface FormLayoutOptions<
+    T extends FormLayoutType | `${FormLayoutType}` = FormLayoutType.Default,
```

Comment:
Not sure if I understand this change and the ones for the grd/row layout options :thinking:

Reply:
This typing change is a side effect of us using `enum` [A video showing why we shouldn't use enums in TypeScript](https://www.youtube.com/watch?v=jjMbPt_H3RQ)

So basically `` `${FormLayoutType}` `` turns an enum type to a type with a union of the values of the enum.
So you can in this case write `FormLayoutType.Default` OR `'default'` and both are valid. Which I think makes a lot more sense, AND Also you get intellisense of the types instead of needing to import the enum type and use that.

Reply:
Yes, enums are weird in typescript and we should probably not use them if we can avoid it :smile: 
I think we started using object literals in some other places as an alternative, e.g.

```ts
export const Environment = {
    development: 'dev',
    production: 'prod',
    testing: 'test',
} as const;

export type Environment = (typeof Environment)[keyof typeof Environment];
```

Reply:
Sweet, but I don't want to change the existing enum. But good for future reference

Reply:
Yeah that's out of scope here :smile: